api/controllers/v0.3/DependentAccountAPI.js

```diff
@@ -52,21 +71,34 @@ const DependentAccountAPI = (args) => {
       const canCreateDependents = checkDependentLimit(patron.varFields);
 
       if (!canCreateDependents) {
-        response["eligible"] = false;
-        response["description"] =
-          "This patron has reached the limit to create dependent accounts.";
+        throw new NotEligibleCard(
+          "You have reached the limit of dependent cards you can receive via online application."
+        );
       } else {
         response["eligible"] = true;
```

Comment:
Is there ever a situation where `response["eligible"] == false` ?  I have logic in the client that handles that, but if this is impossible I can simplify and remove.

Reply:
There won't be a situation like that anymore. If the patron is not eligible, an error will be thrown instead of returning a response object that includes `eligible: false`. I think it makes it easier if you call the API and get a 400 with bad data, rather than getting a 200 and having to parse it to see what the eligibility is.

Reply:
that makes sense to me too. In this case then I can remove parsing of the successful response which essentially is like a 204 for me